scripts/get-usage/deduplicate-components.ts

```diff
@@ -0,0 +1,42 @@
+const componentNameChanges: Record<string, string | undefined> = {
+	'@guardian/src-ed-button/Button':
+		'@guardian/source-react-components-development-kitchen/EditorialButton',
+	'@guardian/src-ed-button/LinkButton':
+		'@guardian/source-react-components-development-kitchen/EditorialLinkButton',
+};
+
+export const deduplicateComponents = (components: string[]): string[] => {
```

Comment:
I wonder if we could come up with a more precise name for this function? Something like `groupSrcAndSourceEquivalentComponents`